app/controllers/fitter_happier/fitter_happier_controller.rb

```diff
@@ -0,0 +1,30 @@
+module FitterHappier
+  class FitterHappierController < ActionController::Base
+    layout nil
+
+    def index
+      render(:plain => "FitterHappier Site Check Passed\n")
+    end
+
+    def site_check
+      time = Time.now.to_formatted_s(:rfc822)
+      render(:plain => "FitterHappier Site Check Passed @ #{time}\n")
+    end
+
+    def site_and_database_check
+      version = DatabaseCheck.schema_version
+      time = Time.now.to_formatted_s(:rfc822)
+      render(:plain => "FitterHappier Site and Database Check Passed @ #{time}\nSchema Version: #{version}\n")
+    end
+
+    private
+
+    def process_with_silence(*args)
```

Comment:
They are aliasing [`process`](http://apidock.com/rails/AbstractController/Base/process), to silence the logger. This method will be called by Rails. 
I think we can replace `process_with_silence` and the `alias_method_chain` call with an [`around_action`](http://guides.rubyonrails.org/action_controller_overview.html#after-filters-and-around-filters)